SwitchDemo/README.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Xamarin.Android - Switch Demo
+description: "Shows how to use a switch control (Android Ice Cream Sandwich)"
+page_type: sample
+languages:
+- csharp
+products:
+- xamarin
+extensions:
+    tags:
+    - androidicecreamsandwich
```

Comment:
again, ice cream sandwich is old news! But let's add `ui` since the metadata shows a tag for "User Interface"

```suggestion
    - ui
```